pkg/clusteragent/admission/mutate/test_utils.go

```diff
@@ -133,3 +137,8 @@ func withContainer(pod *corev1.Pod, nameSuffix string) *corev1.Pod {
 	pod.Spec.Containers = append(pod.Spec.Containers, corev1.Container{Name: pod.Name + nameSuffix})
 	return pod
 }
+
+func withNamespace(pod *corev1.Pod, namespace string) *corev1.Pod {
```

Comment:
Is this function going to be used again? Or can it just be rolled into fakePodWithNamespaceAndLabel?